optimum/onnxruntime/modeling_decoder.py

```diff
@@ -14,25 +14,26 @@
 """Classes handling causal-lm related architectures in ONNX Runtime."""
 
 import logging
-import os
+import re
 import shutil
 from pathlib import Path
-from typing import Any, Dict, Optional, Tuple, Union
+from tempfile import TemporaryDirectory
+from typing import Any, Dict, List, Optional, Tuple, Union
 
 import numpy as np
 import torch
 import transformers
 from transformers import AutoModelForCausalLM, PretrainedConfig
```

Comment:
if `PretrainedConfig` is imported only when doing type checking, we can add

```
from typing import TYPE_CHECKING

if TYPECHECKING:
    from transformers import PretrainedConfig
```

Reply:
Done!